### PR TITLE
CDPT-2015 Add probation area field to case details.

### PR DESCRIPTION
**Pull request by njpreddy**
## Self-review checklist

* [ ] (1) Quick stakeholder demo done OR
* [ ] (2) ...bug with before and after screenshots
* [ ] (3) Tests passing
* [ ] (4) Branch ready to be merged (not work in progress)
* [ ] (5) No superfluous changes in diff
* [ ] (6) No TODO's without new ticket numbers
* [ ] (7) PR Prefixed with ticket number e.g. `CT-7654 ...`
* [ ] (8) Data migration script is created if any of letter templates is changed

### Related JIRA tickets

https://dsdmoj.atlassian.net/jira/software/c/projects/CDPT/boards/1152?assignee=557058%3A2d2f4a33-f427-483f-9c36-338caee60eae&selectedIssue=CDPT-2015

### CODE REVIEW

File: app/models/case/sar/offender.rb [hunk 1]
```diff
@@ -56,6 +56,7 @@ def searchable_fields_and_ranks
         subject_aliases: "B",
         third_party_company_name: "B",
         third_party_name: "B",
+        probation_area: "B",
```
Comment (vertism) [inline app/models/case/sar/offender.rb:59]: Does this need to be made searchable?

File: spec/services/retention_schedules/anonymise_case_service_spec.rb [hunk 2]
```diff
@@ -68,6 +68,7 @@
       postal_address: "Anon address",
       previous_case_numbers: "XXXXXXXX",
       prison_number: "123456",
+      probation_area: "Smallville",
```
Comment (vertism) [inline spec/services/retention_schedules/anonymise_case_service_spec.rb:71]: Should this be anonymised?

File: app/models/case/sar/offender.rb (hunk 1 above)
Comment (njpreddy) [reply app/models/case/sar/offender.rb]: I've removed probation area from searchability.

File: spec/services/retention_schedules/anonymise_case_service_spec.rb (hunk 2 above)
Comment (njpreddy) [reply spec/services/retention_schedules/anonymise_case_service_spec.rb]: Probation area is now not anonymised.

**Review by vertism — APPROVED** [review]
LGTM

**Pull request merged**